sessionctx/variable/tidb_vars.go

```diff
@@ -453,6 +453,9 @@ const (
 
 	// TiDBEnableAmendPessimisticTxn indicates if amend pessimistic transactions is enabled.
 	TiDBEnableAmendPessimisticTxn = "tidb_enable_amend_pessimistic_txn"
+
+	// TiDBEnableEnumLengthLimit indicates the enum/set element length should be shorter than 255 is enabled.
+	TiDBEnableEnumLengthLimit = "tidb_enable_enum_length_limit"
```

Comment:
Please use config instead of the session variable. Like https://github.com/pingcap/tidb/pull/15012/files